### DISCUSSION TITLE
How do I get the Badge?

### DISCUSSION THREAD

**Discussion by Rough4993** [community/community#68176] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/145235260/b2a26223-7f45-4257-ab16-8994efe36c22)![image](https://github.com/community/community/assets/145235260/a3a0df68-f69d-4da2-9c3e-0c0f2b695f37)

**Comment by Durgesh4993** [comment]
# Heart On Your Sleeve
React to something on GitHub with a ❤️ emoji

# Open Sourcerer
User had PRs merged in multiple public repositories

**Comment by Durgesh4993** [comment]
Hey @Rough4993 , the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges are not yet released to public GitHub users at the moment.

Please kindly wait for & stay tuned to the official news release by GitHub on their blog over [here](https://github.blog/).

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information (based on my understanding) on the mechanics to achieve those two new badges.

Hope this clarifies & answers your question, take care Samuel

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
